### PR TITLE
[ISSUE #771]Use url parameters instead of special configuration

### PR DESCRIPTION
**Pull request by haoyann**
The branch of 0.2.0 is master-0.2.0. If your PR is to solve the problem of 0.2.0, please submit the PR to it.
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

#771

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist.
* [ ] Run `mvn clean compile --batch-mode -DskipTests=false -Dcheckstyle.skip=false -Drat.skip=false -Dmaven.javadoc.skip=true to make sure basic checks pass.

### CODE REVIEW

**Review by KeRan213539 — CHANGES_REQUESTED** [review]
Please do not remove the configuration method directly. Add the method of getting from URL on the basis of retaining the configuration. If both are set, the URL shall prevail.
Add a description in the configuration file, recommend the use of URL mode, and explain that the configuration mode will be deleted later

**Comment by haoyann** [comment]
> Please do not remove the configuration method directly. Add the method of getting from URL on the basis of retaining the configuration. If both are set, the URL shall prevail.
> Add a description in the configuration file, recommend the use of URL mode, and explain that the configuration mode will be deleted later

Get it.

File: dubbo-admin-server/src/main/resources/application.properties [hunk 1]
```diff
@@ -15,35 +15,42 @@
...
-#namespace used by nacos
-admin.registry.namespace=public
-admin.config-center.namespace=public
-admin.metadata-report.namespace=public
 
-admin.apollo.token=e16e5cd903fd0c97a116c873b448544b9d086de9
-admin.apollo.appId=test
-admin.apollo.env=dev
-admin.apollo.cluster=default
-admin.apollo.namespace=dubbo
```
Comment (KeRan213539) [inline dubbo-admin-server/src/main/resources/application.properties:46]: These configurations are reserved and commented out for the time being, and add a note like this: "not recommended, later versions will be deleted."

**Review by KeRan213539 — CHANGES_REQUESTED** [review]
Old configurations are reserved and commented out for the time being, and add a note like this: "not recommended, later versions will be deleted."

**Review by KeRan213539 — APPROVED** [review]

**Pull request merged**